spec/migration_spec.cr

```diff
@@ -26,4 +42,28 @@ describe LuckyMigrator::Migration::V1 do
     end
     exists.should be_false
   end
+
+  describe "statement execution order" do
+    Spec.after_each do
+      MigrationWithOrderDependentExecute::V998.new.down(log: false)
+    end
+
+    it "runs execute statements in the order they were called" do
+      MigrationWithOrderDependentExecute::V998.new.up(log: false)
+      columns = get_column_names("execution_order")
+      columns.includes?("new_col").should be_true
+      columns.includes?("bar").should be_true
+    end
+  end
+end
+
+def get_column_names(table_name)
```

Comment:
Could you add `private` to this so it isn't exposed to other tests?

Reply:
Sure